### DISCUSSION TITLE
【喜报】新艾利都反诈中心上线啦！

### DISCUSSION THREAD

**Discussion by user1-cloud** [share121/inter-knot#1379] [其他]
![c9b33eafd18b48e47292ca455b53780b737b74c4](https://github.com/user-attachments/assets/94c3a930-96c1-4c0d-9e65-3423d6e7ca63)
# <div align="center">今日，新艾利都反诈中心正式上线！</div>

新艾利都反诈中心是治安局打击治理绳网新型违法犯罪工作部际联席会议合成作战平台，集**资源整合、情报研判、侦查指挥**为一体，在**打击、防范、治理电信网络诈骗等新型违法犯罪**中发挥着重要作用。

***

### <div align="center">成立背景</div>
近来，以绳网诈骗为代表的新型网络违法犯罪形势**严峻复杂**，发案数仍在**高位运行**，骗术手法**翻新极快**、**隐蔽性更强**，严重侵犯市民们的**财产权利和人身权利**，已成为影响市民们**获得感、幸福感、安全感**的突出问题。新艾利都治安局打击治理实践证明，绳网诈骗犯罪是**可防性犯罪**，**宣传防范**是压降发案的重要手段。

### <div align="center">软件特色</div>
 - 这是款能有效帮助我们预防各种诈骗一款软件，能够帮助用户**主动识别**哪些绳网帖子是诈骗信息。
 - 可以对那些诈骗帖子或信息**快速向平台举报**，帮助他人减少遇到类似的情况。
 - 能够帮助用户**随时监控**各种恶意软件，让各种骗局无路可走，给每个用户带来一个非常安全的生活环境。

***

### <div align="center">软件亮点</div>
 - 可以帮助用户**自动拦截**各种骚扰帖子和信息，对那些疑似诈骗的帖子可以快速进行举报。
 - 平台里每天都有超多的防诈骗信息新闻为用户**免费**提供，用户随时在这里**学习各种防诈骗知识**。

***

### <div align="center">下载地址</div>
|PC端|安卓端|
|:---:|:---:|
|[立即下载](https://www.bilibili.com/video/BV1GJ411x7h7/?share_source=copy_web&vd_source=722b2d6c4a0efb2b0fc1aa3c34f95a9d "点击以下载PC端")|[立即下载](https://www.bilibili.com/video/BV1uT4y1P7CX/?share_source=copy_web&vd_source=722b2d6c4a0efb2b0fc1aa3c34f95a9d "点击以下载安卓端")|

**Comment by su2569** [comment]
治安局提醒您：建议安装国家反诈中心
[![image](https://bkimg.cdn.bcebos.com/pic/6a63f6246b600c3387448165dc1f460fd9f9d72a0b58?x-bce-process=image/format,f_auto/resize,m_lfit,limit_1,h_1000)](https://bkimg.cdn.bcebos.com/pic/6a63f6246b600c3387448165dc1f460fd9f9d72a0b58?x-bce-process=image/format,f_auto/resize,m_lfit,limit_1,h_1000)
https://www.coolapk.com/apk/com.hicorenational.antifraud
https://apps.apple.com/cn/app/%E5%9B%BD%E5%AE%B6%E5%8F%8D%E8%AF%88%E4%B8%AD%E5%BF%83/id1552823102?l=en-GB
https://sj.qq.com/appdetail/com.hicorenational.antifraud?fromcase=40002&from_wxz=1

**Comment by KELERSAMA** [comment]
原来治安局发的反诈宣传帖说的事都是真的 
诈骗犯竟然真的无处不在！！！

**Comment by Prance36** [comment]
不是，治安局进骗子了？

**Comment by LapStdy** [comment]
被诈骗了

**Comment by Bicjet** [comment]
不是哥们，这算诈骗犯反诈骗吗
